packages/storybook-vue/src/stories/Table/Table.stories.mdx

```diff
@@ -0,0 +1,73 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Table } from "@utrecht/component-library-vue";
+import { argTypes } from "./argTypes.ts";
+
+<Meta id="vue-table" title="Vue.js Component/Table" component={Table} argTypes={argTypes} />
+
+# Table component
+
+export const Template = (args) => ({
+  components: { Table },
+  setup() {
+    return { args };
+  },
+  template: `
+  <Table v-bind="args" aria-labelledby="53818ecc-f9a6-4a9d-9e63-12a5ebb89e31">
+  <TableCaption id="53818ecc-f9a6-4a9d-9e63-12a5ebb89e31"> Table Caption </TableCaption>
```

Comment:
`aria-labelledby` should not be necessary for `caption`, it should work automatically. Did you run into issues with the label in the accessibility tree?

The main use for `aria-labelledby` would be if you have a situation where `h1`-`h6` or `figcaption` is preferable to `caption`.